build-tools/create-packs/Directory.Build.targets

```diff
@@ -62,7 +62,41 @@
       DependsOnTargets="CreateAllRuntimePacks" >
     <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Ref.proj&quot;" />
     <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Sdk.proj&quot;" />
-    <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Xamarin.Android.Sdk.Lite.proj&quot;" />
+    <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Workload.proj&quot;" />
+  </Target>
+
+  <Target Name="ExtractWorkloadPacks"
+      DependsOnTargets="GetXAVersionInfo" >
+    <ItemGroup>
+      <_PackFilesToDelete Include="$(DotNetPreviewPath)packs\Microsoft.Android*\**\*.*" />
+      <_WLManifest Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\Microsoft.Android.Workload.$(AndroidPackVersionLong).nupkg" />
+      <_WLPacks Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\Microsoft.Android.Sdk.$(AndroidPackVersionLong).nupkg" />
+      <_WLPacks Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\Microsoft.Android.Ref.$(AndroidPackVersionLong).nupkg" />
+      <!-- Runtime packs are not yet supported by workloads -->
+      <!-- <_WLPacks Include="$(XamarinAndroidSourcePath)bin\Build$(Configuration)\nupkgs\Microsoft.Android.Runtime.$(AndroidPackVersionLong).nupkg" /> -->
+    </ItemGroup>
+    <RemoveDir
+        Directories="%(_PackFilesToDelete.RootDir)%(_PackFilesToDelete.Directory)"
+    />
+    <Unzip
+        SourceFiles="@(_WLManifest)"
+        DestinationFolder="$(DotNetPreviewPath)sdk-manifests\$(DotNetPreviewVersionBand)\Microsoft.Android.Workload"
+    />
+    <Unzip
+        SourceFiles="@(_WLPacks)"
+        DestinationFolder="$(DotNetPreviewPath)packs\$([System.String]::Copy('%(_WLPacks.Filename)').Replace('.$(AndroidPackVersionLong)', ''))\$(AndroidPackVersionLong)"
+    />
+    <ItemGroup>
+      <_UnixExecutables Include="$(DotNetPreviewPath)packs\Microsoft.Android.Sdk\*\tools\$(HostOS)\**\*.*" />
+    </ItemGroup>
+    <Exec
+        Condition="$([MSBuild]::IsOSPlatform('osx'))"
```

Comment:
Shouldn't this also check for linux?

Reply:
Thanks, I updated this to check $(HostOS) for either Darwin or Linux.